.changeset/tender-dots-warn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/admin": major
+---
+
+Remove the `paper` class key from `FilterBarPopoverFilterClassKey`
+Instead of using `styleOverrides` for `paper` in the theme, use the `slotProps` and `sx` props.
```

Comment:
I think there needs to be a line-break between the title and the details. 

```suggestion

Instead of using `styleOverrides` for `paper` in the theme, use the `slotProps` and `sx` props.
```